addons/website_crm_partner_assign/models/res_partner.py

```diff
@@ -67,3 +80,18 @@ def _compute_implemented_partner_count(self):
     def _compute_partner_weight(self):
         for partner in self:
             partner.partner_weight = partner.grade_id.partner_weight if partner.grade_id else 0
+
+    def _compute_opportunity_count(self):
+        super()._compute_opportunity_count()
+        opportunity_data = self.env['crm.lead'].with_context(active_test=False).read_group(
```

Comment:
Why counting lost / archived ?

Reply:
To be consistent with super method. We count the lost leads for customers, so may be we should consider such leads for assigned partners as well.